k8s/celery.yaml.j2

```diff
@@ -37,4 +37,24 @@ spec:
             limits:
               cpu: {{ CELERY_WORKER_CPU_LIMIT }}
               memory: {{ CELERY_WORKER_MEMORY_LIMIT }}
+        - name: {{ CELERY_SCHEDULER_NAME }}
+          image: "{{ APP_IMAGE }}:{{ APP_IMAGE_TAG }}"
+          imagePullPolicy: {{ APP_IMAGE_PULL_POLICY }}
+          volumeMounts:
+            - name: developer-portal-fs
+              mountPath: {{ APP_MOUNT_PATH }}
```

Comment:
Do we still need this since we hosts our media under S3 now?

Reply:
Celerybeat needs somewhere to use as a scheduler database of jobs. We could use django-celery-scheduler, which is a Django ORM backend for it, but then that means we [re]configure jobs manually, or via data migrations, so i went with the default scheduler, which is a simple on-disk instance of the python [shelve](https://docs.python.org/3/library/shelve.html) "database". So, if we're going to have more than one node/pod running celery-beat, it should be on a shared FS. If we're definitely going to cap it at one, then I think we'd be OK with a non-persistent disk